Atomic/trust.py

```diff
@@ -29,10 +28,16 @@ def cli(subparser):
                      atomic: openshift-based atomic registry"""
     commonp.add_argument("-k", "--pubkeys", nargs='?', default=[],
                          action="append", dest="pubkeys",
-                         help=_("Absolute path of installed public key(s) to trust for TARGET. "
+                         help=_("Local path or URL of public key(s) to trust for TARGET. "
+                                "Keys are parsed and encoded into policy.json. "
+                                "May used multiple times to define multiple public keys. "
+                                "File(s) must exist before using this command."))
+    commonp.add_argument("-f", "--pubkeysfile", nargs='?', default=[],
```

Comment:
Shoudl --pubkeys and --pubkeysfile be mutually exlusive? Or would a user specify both on the same command line?